test/ProgressOnderwijsUtils.Tests/NonNullableNullabilityCheck.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+using ProgressOnderwijsUtils.Data;
+
+namespace ProgressOnderwijsUtils.Tests;
+
+#nullable enable
```

Comment:
Ik dacht dat dit de default was, toch?

Reply:
(in onze csproj dan, niet globaal overal)

Kun je checken of het weghalen hiervan iets doet?